### PR TITLE
ramiray/c01-aws01

### PR DESCRIPTION
**Pull request by ramiray**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-aws01/ramiray/readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Task1 :
+1. Created the instance with the following command in the AWS CLI.
+
```
Comment (caiocezart) [inline classes/01class/exercises/c01-aws01/ramiray/readme.md:3]: can you add the commands used to create the security group and rules?

File: classes/01class/exercises/c01-aws01/ramiray/readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+aws ec2 run-instances --image-id  ami-088ff0e3bde7b3fdf --count 1 --instance-type t2.micro --key-name Devopsacademy --security-group-ids sg-0449aeda785be7dd2 --subnet-id subnet-0f9f7c47
+
+```
+2. From the terminal , used the ssh command to connect the instance with the following command.
```
Comment (caiocezart) [inline classes/01class/exercises/c01-aws01/ramiray/readme.md:8]: can you add commands used to create the ec2 pem key?

File: classes/01class/exercises/c01-aws01/ramiray/readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+
+Task2: 
+
+1. Created the instance with the following command in the AWS CLI.
+
+```
+
+aws ec2 run-instances --image-id  ami-088ff0e3bde7b3fdf --count 1 --instance-type t2.micro --key-name Devopsacademy --security-group-ids sg-0449aeda785be7dd2 --subnet-id subnet-0f9f7c47 --no-associate-public-ip-address
```
Comment (caiocezart) [inline classes/01class/exercises/c01-aws01/ramiray/readme.md:21]: add commands used to create security group, rules and pem file

File: classes/01class/exercises/c01-aws01/ramiray/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+Task1 :
+ 
+1. Command to create security Group and adding the rules to the SG. 
+
+```
+aws ec2 create-security-group --group-name Devopsacademy-sg --description "My security group" --vpc-id vpc-7fd6d818
```
Comment (roperto) [inline classes/01class/exercises/c01-aws01/ramiray/README.md:6]: The exercise said you could use the "default VPC", which means you did not need to specify the `--vpc-id` if that is the default VPC.

That said, I usually prefer to do like you did -- explicitly setting the VPC -- I think it is safer like you did.

**Pull request merged**